examples/src/main/java/kafka/examples/Consumer.java

```diff
@@ -83,13 +83,4 @@ public void doWork() {
         }
     }
 
-    @Override
-    public String name() {
```

Comment:
Let's file a ticket in JIRA for removing the dependency on `ShutdownableThread` - our examples should not rely on internal apis.

Reply:
+1  "examples" module should not be dependant on internal server side components, filed https://issues.apache.org/jira/browse/KAFKA-14708